### ISSUE TITLE
Reduced plasma acylcarnitine levels

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5958]
**Preferred term label:** Reduced plasma acylcarnitine levels

**Synonyms**

**Definition (free text, please give PubMed ID)** A decreased concentration of acylcarnitine in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
